app/controllers/home_controller.rb

```diff
@@ -36,4 +37,22 @@ def show
 
   end
 
+  private
+  # Checks for the presence of mandatory organisation parameters,
+  # returning false if any are not present and true if all are
+  # present
+  def complete_organisation_details?
+
+    return [
```

Comment:
Very minor but I don't think we need `return` here. Ruby will by default return the last statement.
```suggestion
    [
```